opentelemetry-api/opentelemetry/__init__.py

```diff
@@ -0,0 +1 @@
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
```

Comment:
Since we don't support Python 2 anyway (and probably not <3.3 too), why use the pkgutil style and not the implicit, native style for namespace packages? https://packaging.python.org/guides/packaging-namespace-packages/#native-namespace-packages

Caveat: I've never created a namespace package before, so this is just me reading the docs and wondering.

Reply:
Good point, I'll update it in my next PR.